### PR TITLE
Update for five digit team numbers

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

**Comment by jasondaming** [comment]
I suggested to modify the traditional TE.AM to XTE.AM, what do you think? @sciencewhiz 
https://www.chiefdelphi.com/t/team-numbers-have-officially-hit-5-digits-what-now/466213/24

**Comment by PeterJohnson** [comment]
I liked the suggestion of `10.STE.AM.x` later in the thread.

**Comment by jasondaming** [comment]
I was worried that would change the "obviousness" that it was the team number.  I do think that would prevent confusion around `10.XTE.AM.X` which less ideally uses X twice (we could also change that variable, we already use Y and Z in other places).  Either way I think both of these are much better than just TE.AM

**Comment by sciencewhiz** [comment]
I don't like STE.AM, it uses something unrelated in a confusing way. 

I think TE.AM with explanation is sufficient, but if I were to choose something else it'd prefer TEA.M#

**Comment by PeterJohnson** [comment]
I think just sticking with TE.AM works too with a simple explanation that "AM" is always the last two digits of the team number, so "TE" is 3 digits for 5-digit teams.

File: source/docs/networking/networking-introduction/ip-configurations.rst [hunk 1]
```diff
@@ -6,14 +6,18 @@ IP Configurations
 TE.AM IP Notation
 -----------------
 
-The notation TE.AM is used as part of IPs in numerous places in this document. This notation refers to splitting your four digit team number into two digit pairs for the IP address octets.
+The notation TE.AM is used as part of IPs in numerous places in this document. This notation refers to splitting your five digit team number into digits for the IP address octets. where AM is the last two digits of the team number, and TE is the first three digits. Leading zeros are optional. This scheme supports team numbers up to 25599.
```
Comment (jasondaming) [inline source/docs/networking/networking-introduction/ip-configurations.rst:9]:
```suggestion
The notation TE.AM is used as part of IPs in numerous places in this document. This notation refers to splitting your five digit team number into digits for the IP address octets. Where AM is the last two digits of the team number, and TE is the first three digits. Leading zeros are optional. This scheme supports team numbers up to 25599.
```

**Review by jasondaming — APPROVED** [review]

**Pull request merged**